### PR TITLE
PHP SDK: update documentation about updates

### PR DESCRIPTION
**Pull request by roxblnfk**
PHP Updates:
- sync headers with Java SDK;
- add docs about dynamic registering of Update handlers

- [x] Waiting for [PHP SDK](https://github.com/temporalio/sdk-php) 2.11

### CODE REVIEW

**Comment by roxblnfk** [comment]
Ready to merge 👍

**Review by fairlydurable — APPROVED** [review]
Unblocking this for merge.

Talked with Brian about content, and it should be good to go.
Did not find issues that would prevent this moving forward.

Thank you for bringing this Update content to PHP.

File: docs/develop/php/message-passing.mdx [hunk 1]
```diff
@@ -2,9 +2,13 @@
 id: message-passing
 title: Workflow message passing - PHP SDK
 sidebar_label: Messages
+description: Learn how to develop with Signals, Queries, and Updates using the Temporal PHP SDK. Define, handle, and send Signals or Queries, and validate updates from a Temporal Client.
```
Comment (fairlydurable) [inline docs/develop/php/message-passing.mdx:5]:
```suggestion
```

**Comment by fairlydurable** [comment]
Please note, to make this deploy successfully, I had to remove the duplicate description and I was only able to remove the added one (in the standard place) and leave the original (in the non-standard place). When we audit front matter, this should be corrected.

**Pull request merged**